views/modules.pug

```diff
@@ -2,39 +2,13 @@ extends layout
 
 block content
   .page-header
-    h3 New Module
-
-  table#plants.table-striped.table-bordered.table-sm
```

Comment:
Removing the whole table as per https://urbanrivers.slack.com/archives/C015RUE8ZQB/p1595367858015000